perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/pages/AppsView.java

```diff
@@ -57,34 +71,76 @@ public void refresh(ClickEvent event) {
 		draw();
 	}
 
+	@UiHandler(value = "allApps")
```

Comment:
I would add draw() method also on filterButton click.
```suggestion
    @UiHandler(value = "filterButton")
	public void filterButton(ClickEvent event) {
		draw();
	}
	
	@UiHandler(value = "allApps")
```